spec/features/comment_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Comments', type: :feature do
+  before :each do
+    @user = User.create(name: 'User', email: 'user@example.com', password: 'user123')
+    @post = @user.posts.build(content: 'test post', id: 1)
+    @post.save
+  end
```

Comment:
`[Optional]` favor the use of `let` over `before`. Please, check this guide: http://www.betterspecs.org/#let